src/FSharp.AWS.DynamoDB/Expression/ConditionalExpr.fs

```diff
@@ -172,21 +165,21 @@ let extractQueryExpr (recordInfo: RecordTableInfo) (expr: Expr) : ConditionalExp
         invalidArg "expr" "supplied query is not a closed expression."
 
     let invalidQuery () =
-        invalidArg "expr" <| sprintf "Supplied expression is not a valid conditional."
+        invalidArg "expr"
+        <| sprintf "Supplied expression is not a valid conditional."
```

Comment:
isnt there a rule about not doing new lines for only one pipe?
not the end of the world but also not better

Reply:
No, no specific rules around pipes afaik, but I assume fantomas is aware of them to prioritise breaking on them.

This particular rule appears to be `fsharp_max_infix_operator_expression` which was set to 60, I can probably bump all the width numbers up a little further though

Reply:
in general I've weaned myself off using `<|` and just use parens now - this rule would defintiely cure me completely!

Reply:
I’ve set `fsharp_max_infix_operator_expression` to 100 and it’s picking up much fewer of these. I think it’s a tradeoff between splitting piped expressions across multiple lines where the flow is easier to follow, vs getting it done in a one-liner which may make the outer context a bit cleaner.

Agree with backward pipes, I rarely use them